net/interfaces/interfaces_darwin.go

```diff
@@ -17,3 +22,70 @@ func fetchRoutingTable() (rib []byte, err error) {
 func parseRoutingTable(rib []byte) ([]route.Message, error) {
 	return route.ParseRIB(syscall.NET_RT_IFLIST2, rib)
 }
+
+var (
+	mu      sync.Mutex
```

Comment:
name "mu" is too short for its large package-level scope. How about `ifNamesMu`? Or the ol':

```go
var ifNames struct {
     sync.Mutex
     m map[int]string // ifindex => name
}
```

Then `ifNames.Lock`, etc. And use `mak.Set` to init it probably.

Reply:
Will switch to a struct.